src/sass/_variables.scss

```diff
@@ -17,7 +17,7 @@ $post-info-text-color: #676767;
 $old-price-color: #a5a5a5;
 
 $form-border-color: #292929;
-
+$flkty-bg-color: white;
```

Comment:
ten sposób deklarowania kolorów to jedna ze złych praktyk. Popraw na prawidłowy.